benchmark/utils/loggers.hpp

```diff
@@ -172,21 +172,13 @@ template <typename ValueType>
 struct ResidualLogger : gko::log::Logger {
     using rc_vtype = gko::remove_complex<ValueType>;
 
-    // TODO2.0: Remove when deprecating simple overload
-    void on_iteration_complete(const gko::LinOp* solver,
-                               const gko::size_type& it,
+    void on_iteration_complete(const gko::LinOp*, const gko::size_type&,
```

Comment:
Isnt this technically interface breaking ?

Reply:
I don't think we care about interface changes in our benchmarks.

Reply:
:+1: I missed that this was in the benchmarks.